### PR TITLE
Feature/english copy

### PR DESCRIPTION
**Pull request by jahlela**
Copy editing and updated translations for all feature pages, and pricing.

### CODE REVIEW

File: src/pages/features/captable.jsx [hunk 1]
```diff
@@ -76,16 +76,16 @@ export default withI18n()(({ i18n, ...props }: Props) => (
...
           </Trans>,
           <Trans>
             Supports issuance, transfer, convertibles, stock split, valuations, treasury shares,
             unlimited share classes and more
           </Trans>,
           <Trans>
-            Analyze it by stakeholder, share class, stakeholder group and view it with all,
-            outstanding or diluted shares
+            Analyze your cap table by stakeholder, share class, or stakeholder group, and view it
+            with outstanding or diluted shares
```
Comment (ovcOS) [inline src/pages/features/captable.jsx]: Shouldn't delete 'all'. It's one of the view modes (ownership mode, meaning outstanding + treasury)

File: src/pages/features/captable.jsx [hunk 2]
```diff
@@ -100,8 +100,8 @@ export default withI18n()(({ i18n, ...props }: Props) => (
         features={[
           <Trans>In some countries numbering shares is necessary, and it can be a real pain</Trans>,
           <Trans>
-            Auto-assign share numbers for all your transactions; Ledgy will determine the next
-            available ones
+            Auto-assign share numbers for all your transactions. Ledgy will determine the next
+            available numbers
```
Comment (ovcOS) [inline src/pages/features/captable.jsx]: I personally preferred it how it was before 😇

File: src/pages/features/modeling.jsx [hunk 3]
```diff
@@ -82,7 +82,7 @@ export default withI18n()(({ i18n, ...props }: Props) => (
           <Trans>
             See how many shares have to be reserved for a fixed percentage employee pool
           </Trans>,
-          <Trans>Use pre- or post-money valuation and see the respective other</Trans>
+          <Trans>Choose between pre- or post-money valuation</Trans>
```
Comment (ovcOS) [inline src/pages/features/modeling.jsx]: It was already wrong before, but it should be 'Choose between pre- AND post-money valuation'. It'd be 'or' if it was 'choose either this or that'

File: src/pages/features.jsx [hunk 4]
```diff
@@ -33,11 +33,11 @@ export default withI18n()(({ i18n, ...props }: Props) => (
           <Feature {...props} name={i18n.t`Employee Participation Plans`} url="esop">
             <ul className="pl-0 pt-2">
               <FeatureLi>
-                <Trans>Save hours of work by getting rid of manual spreadsheet processes</Trans>
+                <Trans>Save hours of manual work by getting rid of complicated spreadsheets</Trans>
               </FeatureLi>
               <FeatureLi>
                 <Trans>
-                  Supports everything from pools to options, phantom options, warrants, vested
+                  Supports everything from pools to options, to phantom options, warrants, vested
```
Comment (ovcOS) [inline src/pages/features.jsx]: This phrase was originally a bit confusing already. I'd suggest something simpler: Supports everything: pools, options, phantom options, warrants, vested stock, and inverse vesting

File: src/pages/features.jsx [hunk 5]
```diff
@@ -58,18 +58,18 @@ export default withI18n()(({ i18n, ...props }: Props) => (
...
-                  split, share destruction, valuations
+                  Support for many transaction types, including issuance, transfer, convertibles,
+                  stock split, share destruction, and valuations
                 </Trans>
               </FeatureLi>
               <FeatureLi>
                 <Trans>
                   Featuring unlimited share classes, treasury shares, pooled investments, and
-                  automatic share numbering
+                  automatic numbered shares
```
Comment (ovcOS) [inline src/pages/features.jsx]: Getting a bit picky here, but stating it like implies that shares are automatic, whereas it's numbering that's automatic. If 'automatic share numbering' sounds funny, then maybe 'automatically numbered shares' would be more grammatically correct.

File: src/pages/features/collaboration.jsx [hunk 6]
```diff
@@ -83,11 +83,12 @@ export default withI18n()(({ i18n, ...props }: Props) => (
             <strong>View</strong> See all cap table info in a read-only mode for due diligences
           </Trans>,
           <Trans>
-            <strong>Portfolio</strong> See only the own stake in their{' '}
+            <strong>Portfolio</strong> Only see only their own
```
Comment (ovcOS) [inline src/pages/features/collaboration.jsx]: 🧐

File: src/pages/features/modeling.jsx [hunk 7]
```diff
@@ -111,8 +111,8 @@ export default withI18n()(({ i18n, ...props }: Props) => (
         header={<Trans>Adjust your simulations</Trans>}
         features={[
           <Trans>
-            Play around with founder- or investor-friendly settings, let convertibles and pools
-            dilute all or only the existing shareholders
+            Play around with founder- and investor-friendly settings. Let convertibles and pools
+            dilute for all potential shareholders or for only the existing shareholders
```
Comment (ovcOS) [inline src/pages/features/modeling.jsx]:
```suggestion
            dilute for all potential stakeholders or for only the existing ones
```

File: src/pages/features/captable.jsx (hunk 1 above)
Comment (jahlela) [reply src/pages/features/captable.jsx]: Ah, good to know. I thought it was an errant comma

File: src/pages/features/captable.jsx (hunk 2 above)
Comment (jahlela) [reply src/pages/features/captable.jsx]: How about `Ledgy will determine the next ones available` ?

File: src/pages/features.jsx (hunk 4 above)
Comment (jahlela) [reply src/pages/features.jsx]: Sounds good. I think I stared at it too long. Actually wound up reading this rant from a NYT editor about improper use of ranges: https://afterdeadline.blogs.nytimes.com/2010/08/24/everything-from-this-to-that/

We'll go with your solution, and consider Philip satisfied 😂

File: src/pages/features/modeling.jsx (hunk 3 above)
Comment (jahlela) [reply src/pages/features/modeling.jsx]: Is it not a choice of one or the other? I don't have a strong preference for `and` or `or` here - both seem fine. I can switch to `and` if you like

File: src/pages/features.jsx (hunk 5 above)
Comment (jahlela) [reply src/pages/features.jsx]: I like `automatically numbered shares`

File: src/pages/features/modeling.jsx (hunk 7 above)
Comment (jahlela) [reply src/pages/features/modeling.jsx]: Are we making a general move to `stakeholders` from `shareholders`?

File: src/pages/features/collaboration.jsx (hunk 6 above)
Comment (jahlela) [reply src/pages/features/collaboration.jsx]: :D Never copyedit with watery eyes!!

File: src/pages/features/captable.jsx (hunk 2 above)
Comment (ovcOS) [reply src/pages/features/captable.jsx]: Sounds good :)

File: src/pages/features/modeling.jsx (hunk 3 above)
Comment (ovcOS) [reply src/pages/features/modeling.jsx]: It is one or the other, but I think that the conjunction after 'choose between' should be 'and'. Your are the native though, if it sounds good to you, all good.

File: src/pages/features.jsx (hunk 5 above)
Comment (ovcOS) [reply src/pages/features.jsx]: 👍

File: src/pages/features.jsx (hunk 4 above)
Comment (ovcOS) [reply src/pages/features.jsx]: 👌😁

File: src/pages/features/collaboration.jsx (hunk 6 above)
Comment (ovcOS) [reply src/pages/features/collaboration.jsx]: Hahaha

File: src/pages/features/modeling.jsx (hunk 7 above)
Comment (ovcOS) [reply src/pages/features/modeling.jsx]: In the app it's stakeholders everywhere, so yeah, just to make it consistent I'd have `stakeholders`

File: src/pages/features/captable.jsx (hunk 1 above)
Comment (jahlela) [reply src/pages/features/captable.jsx]: I'm moving the `all` to the end of the list, so it's obvious it doesn't belong to the outstanding

**Pull request merged**